src/datasources/fault-ds/datasource.js

```diff
@@ -192,6 +192,7 @@ export class OpenNMSFMDatasource {
             "Suppressed Time", "Suppressed Until", "Suppressed By", "IP Address",
             "First Event Time", "Last Event ID", "Last Event Time", "Last Event Source",
             "Last Event Creation Time", "Last Event Severity", "Last Event Label", "Last Event Location",
+            "RelatedAlarms",
```

Comment:
I think we can skip adding this to column to the row - the related alarms don't render well in a table cell.

Reply:
done.